views/includes/page-templates/managing-officer-corporate.html

```diff
@@ -79,13 +79,8 @@ <h2 class="govuk-heading-l">Who can we contact about this managing officer?</h2>
 
       {% include "includes/save-and-continue-button.html" %}
 
-      {% if pageParams.isRegistration %}
-        {% set removePathURL = OE_CONFIGS.MANAGING_OFFICER_CORPORATE_URL + OE_CONFIGS.REMOVE + "/" + id %}
-      {% else %}
-        {% set removePathURL = OE_CONFIGS.UPDATE_MANAGING_OFFICER_CORPORATE_URL + OE_CONFIGS.REMOVE + "/" + id %}
-      {% endif %}
+      {% set removePathURL = OE_CONFIGS.MANAGING_OFFICER_CORPORATE_URL + OE_CONFIGS.REMOVE + "/" + id %}    
```

Comment:
Can move the assignment under the entity_number check and only do it if we're including remove button